worldedit-core/src/main/java/com/fastasyncworldedit/core/internal/command/ConfirmHandler.java

```diff
@@ -1,26 +1,7 @@
-/*
- * WorldEdit, a Minecraft world manipulation toolkit
- * Copyright (C) sk89q <http://www.sk89q.com>
- * Copyright (C) WorldEdit team and contributors
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <https://www.gnu.org/licenses/>.
- */
-
-package com.sk89q.worldedit.internal.command;
+package com.fastasyncworldedit.core.internal.command;
 
 import com.fastasyncworldedit.core.configuration.Settings;
-import com.sk89q.worldedit.command.util.annotation.Confirm;
+import com.fastasyncworldedit.core.util.annotation.Confirm;
```

Comment:
Same again...

Reply:
This one for sure will cause confusion and make it harder to traverse.